### PR TITLE
fix(docker): revert self host volumes change

### PR DESCRIPTION
**Pull request by soedirgo**
It doesn't work OOTB after https://github.com/supabase/supabase/commit/6bc7fdc854713a4c0ce9967c48a02b6b0d5cc40f:
```
Error response from daemon: failed to mount local volume: mount ./volumes/db/data:/var/lib/docker/volumes/docker_volume_database/_data, flags: 0x1000: no such file or directory
```

Also updated images.

### CODE REVIEW

File: docker/.gitignore [hunk 1]
```diff
@@ -1,6 +1,4 @@
-volumes/db/data/**/*
-!volumes/db/data/.gitkeep
```
Comment (soedirgo) [inline docker/.gitignore:2]: nested directories are automatically created for volumes, so no need to `.gitkeep` them

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -115,12 +113,10 @@ services:
...
     volumes:
-      - type: volume
-        source: volume_storage
-        target: /var/lib/storage
+      - ./volumes/storage:/var/lib/storage
 
-  meta:
-    container_name: supabase-meta
+  pg-meta:
+    container_name: supabase-pg-meta
```
Comment (soedirgo) [inline docker/docker-compose.yml:119]: Kong uses `pg-meta` so reverting this too, alternatively we can change it on `kong.yml`

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -139,25 +135,6 @@ services:
       - ${POSTGRES_PORT}:5432
     environment:
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
-      PGDATA: /var/lib/postgresql/data/pgdata
```
Comment (soedirgo) [inline docker/docker-compose.yml:142]: default `PGDATA` when not supplied is `/var/lib/postgresql/data`

**Comment by kiwicopple** [comment]
most of these are fixed by this PR: https://github.com/supabase/supabase/pull/3988

**Comment by soedirgo** [comment]
It still has the same issue:
```
Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /host_mnt/Users/soedirgo/repos/supabase/docker/volumes/db/data
```

I'll rebase to master and update the fix

**Comment by soedirgo** [comment]
Updated :+1:

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**